pylearn2/datasets/dataset.py

```diff
@@ -18,7 +18,7 @@ def __iter__(self):
         return self.iterator()
 
```

Comment:
This file is still on the format checker's whitelist, both for docstrings and for PEP8. Can you take it off the whitelist so the formatting will be checked automatically? I don't think I can guarantee PEP8 and NumpyDoc compliance by manually reviewing the file.